.devcontainer.json

```diff
@@ -0,0 +1,48 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
```

Comment:
Either use only tabs or only spaces for indention.

Reply:
Should be spaces now